_posts/2021-07-30-status-update.md

```diff
@@ -0,0 +1,125 @@
+  [timdeluxe]: https://github.com/timdeluxe
+  [puppetlabs-apt-pr-993]: https://github.com/puppetlabs/puppetlabs-apt/pull/993
+  [kenyon]: https://github.com/kenyon
+  [puppetlabs-apt-pr-991]: https://github.com/puppetlabs/puppetlabs-apt/pull/991
+  [johanfleury]: https://github.com/johanfleury
+  [puppetlabs-firewall-pr-998]: https://github.com/puppetlabs/puppetlabs-firewall/pull/998
+  [vchepkov]: https://github.com/vchepkov
+  [puppetlabs-kubernetes-pr-528]: https://github.com/puppetlabs/puppetlabs-kubernetes/pull/528
+  [danifr]: https://github.com/danifr
+  [puppetlabs-postgresql-pr-1282]: https://github.com/puppetlabs/puppetlabs-postgresql/pull/1282
+  [weastur]: https://github.com/weastur
+  [puppetlabs-tomcat-pr-454]: https://github.com/puppetlabs/puppetlabs-tomcat/pull/454
+  [cocker-cc]: https://github.com/cocker-cc
+  [facterdb-pr-175]: https://github.com/voxpupuli/facterdb/pull/175
+  [bastelfreak]: https://github.com/bastelfreak
+  [facterdb-pr-174]: https://github.com/voxpupuli/facterdb/pull/174
+  [maccelf]: https://github.com/maccelf
+  [facterdb-pr-171]: https://github.com/voxpupuli/facterdb/pull/171
+  [lzap]: https://github.com/lzap
+  [rspec-puppet-facts-pr-131]: https://github.com/voxpupuli/rspec-puppet-facts/pull/131
+  [rspec-puppet-facts-pr-130]: https://github.com/voxpupuli/rspec-puppet-facts/pull/130
+  [rspec-puppet-facts-pr-129]: https://github.com/voxpupuli/rspec-puppet-facts/pull/129
+  [iac-pr-286]: https://github.com/puppetlabs/iac/pull/286
+  [dependabot[bot]]: https://github.com/apps/dependabot
+  [litmus-pr-24]: https://github.com/puppetlabs/litmus/pull/24
+  [litmusimage-pr-33]: https://github.com/puppetlabs/litmusimage/pull/33
+  [fuero]: https://github.com/fuero
+  [pdk-templates-pr-441]: https://github.com/puppetlabs/pdk-templates/pull/441
+  [puppet-strings-pr-285]: https://github.com/puppetlabs/puppet-strings/pull/285
+  [scotje]: https://github.com/scotje
+  [puppetlabs_spec_helper-pr-340]: https://github.com/puppetlabs/puppetlabs_spec_helper/pull/340
+
+  ## Puppet.Dsc Stable Release!
+
+  This week we pushed our [Puppet.Dsc PowerShell module][dsc-repo] to the [PowerShell Gallery at version `1.0.0`][dsc-gallery]!
+  This represents more than a year of work bringing us confidence and stability in the auto-generated modules which Puppetize PowerShell DSC Resources.
+  That done, not much _actually_ changed in this release - a _ton_ of added testing and validation, but the features were already very solid.
+  As always we'll continue to take bug reports and address them but our attention turns now to the long-promised automation;
+  we're working through the auto-publishing of PowerShell modules with DSC Resources to the Puppet Forge!
+  Very soon, there will be less than a 24 delay between when a new module (or new version of an existing module) with DSC Resources lands on the PowerShell Gallery and when it arrives, Puppetized, on the Forge!
+
```

Comment:
```suggestion

## `rpsec-puppet`, `puppet-lint` moved to `puppetlabs` namespace

We have moved `puppet-lint`, `rspec-puppet` to the namespace and released `puppet-lint` `2.5.0` and `rspec-puppet` `2.10.0`. 
This will help facilitate more frequent releases of these gems in the future.
```